### PR TITLE
Add Known Issues page

### PR DESCRIPTION
**Pull request by nyechiel**
Closes: https://github.com/submariner-io/submariner-website/issues/342

There are probably more items we need to add, but these should be enough to get started.

Signed-off-by: nyechiel <nyechiel@redhat.com>

### CODE REVIEW

**Review by aswinsuryan — APPROVED** [review]

**Comment by nyechiel** [comment]
@vthapar @aswinsuryan anything about Lighthouse you think we should be documenting here?

File: src/content/operations/known_issues/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+weight = 50
++++
+
+## General
+
+* Minimum supported Kubernetes version is 1.17.
+* Submariner only supports kube-proxy in iptables mode. IPVS is not supported at this time.
+* Clusters deployed with the Calico network plug-in require further configuration to be compatible with Submariner. Please refer to the
+[Calico-specific deployment instructions](../deployment/calico/).
+
```
Comment (LaumiH) [inline src/content/operations/known_issues/_index.en.md:13]: Only CoreDNS is supported out of the box, in KubeDNS the DNS stub domain needs to be added manually, see the GKE docs

**Review by LaumiH — CHANGES_REQUESTED** [review]
CoreDNS is supported automatically, KubeDNS needs manual config

File: src/content/operations/known_issues/_index.en.md (hunk 1 above)
Comment (nyechiel) [reply src/content/operations/known_issues/_index.en.md:15]: Good point, added

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

File: src/content/operations/known_issues/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+[GKE Quickstart Guide](../../getting_started/quickstart/managed_kubernetes/gke/#final-workaround-for-kubedns) for more information.
+* Clusters deployed with the Calico network plug-in require further configuration to be compatible with Submariner. Please refer to the
+[Calico-specific deployment instructions](../deployment/calico/).
+
+## Globalnet
+
+* Globalnet only supports Pod to remote Service connectivity using Global IPs. Pod to Pod connectivity is not supported at this time.
+* Globalnet is not compatible with Headless Services. Only ClusterIP Services are supported at this time.
+* Globalnet annotates every Service in a cluster, whether or not it was exported.
+* The `subctl benchmark` command is not compatible with Globalnet deployments at this time.
```
Comment (sridhargaddam) [inline src/content/operations/known_issues/_index.en.md:21]: @nyechiel Shall we mention that Healthcheck is not supported with Globalnet currently?

**Pull request merged**